images/openldap/contrib/init.ldif

```diff
@@ -251,3 +251,42 @@ cn: extended-group3
 owner: cn=Person3,ou=people,ou=adextended,dc=example,dc=com
 description: Person3's Group
 member: cn=fake
+
+dn: ou=incomplete,ou=groups,ou=rfc2307,dc=example,dc=com
+objectClass: organizationalUnit
+ou: groups
+description: Group entries with member lists containing missing members
+
+dn: cn=group1,ou=groups,ou=incomplete,ou=rfc2307,dc=example,dc=com
+objectClass: groupOfNames
+cn: group1
+owner: cn=Person1,ou=people,ou=rfc2307,dc=example,dc=com
+description: Person1's Group
+member: cn=Person1,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=Person2,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=Person3,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=Person4,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=Person5,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=INVALID,ou=people,ou=rfc2307,dc=example,dc=com
+member: cn=OUTOFSCOPE,ou=people,ou=OUTOFSCOPE,dc=example,dc=com
```

Comment:
one group with missing and outofscope
one group with just missing
one group with just outofscope

that lets us test flags individually later against specific groups if we want